src/Color.purs

```diff
@@ -273,11 +294,22 @@ fromInt m = rgb r g b
         r = (n `shr` 16) .&. 0xff
         n = clamp 0 0xffffff m
 
--- | Convert a `Color` to its hue, saturation, lightness and alpha values. See
+-- | Convert a `Color` to its Hue, Saturation, Lightness and Alpha values. See
 -- | `hsla` for the ranges of each channel.
 toHSLA :: Color -> { h :: Number, s :: Number, l :: Number, a :: Number }
 toHSLA (HSLA h s l a) = { h, s, l, a }
 
+-- | Convert a `Color` to its Hue, Saturation, Value and Alpha values. See
+-- | `hsva` for the ranges of each channel.
+toHSVA :: Color -> {h :: Number, s :: Number, v :: Number, a :: Number}
```

Comment:
Can we please keep the code style consistent here (spaces after `{` and before `}`)?